apps/src/storage/dataBrowser/DataVisualizer.jsx

```diff
@@ -56,24 +56,75 @@ class DataVisualizer extends React.Component {
 
   updateChart = () => {
     const targetDiv = document.getElementById('chart-area');
+    if (!targetDiv) {
+      return;
+    }
     const records =
       Object.keys(this.props.tableRecords).length > 0 &&
       this.props.tableRecords.map(tableRecord => JSON.parse(tableRecord));
-    if (this.state.chartType === 'Bar Chart' && this.state.values) {
-      var chart = new GoogleChart.MaterialBarChart(targetDiv);
-      const chartData = this.aggregateRecordsByColumn(
-        records,
-        this.state.values
-      );
-      chart.drawChart(chartData, [this.state.values, 'count']);
+
+    let chart;
+    let chartData;
+    const columns = [];
+    const options = {};
+    switch (this.state.chartType) {
+      case 'Bar Chart':
+        if (this.state.values) {
+          chart = new GoogleChart.MaterialBarChart(targetDiv);
+          chartData = this.aggregateRecordsByColumn(records, this.state.values);
+          columns.push(this.state.values, 'count');
+        }
+        break;
+      case 'Histogram':
+        if (this.state.values && this.state.bucketSize) {
+          let min = Infinity;
+          let max = -Infinity;
+          records.forEach(record => {
+            let value = record[this.state.values];
+            if (value < min) {
+              min = value;
+            }
+            if (value > max) {
+              max = value;
+            }
+          });
```

Comment:
Why are we computing `min` and `max` and then not using them?  Also, less efficient but _way_ easier to read:
```js
const values = records.map(r => r[this.state.values]);
const min = Math.min(...values);
const max = Math.max(...values);
```

Reply:
Whoops, no longer needed.